lua/astrocommunity/lsp/actions-preview-nvim/init.lua

```diff
@@ -0,0 +1,28 @@
+local keymaps = {
+  ["<Leader>la"] = {
+    function() require("actions-preview").code_actions() end,
+    desc = "LSP code action",
+    cond = "testDocument/codeAction",
+  },
+}
+
+local mappings = {
+  n = keymaps,
+  v = keymaps,
+}
```

Comment:
We should do this similar to other plugins with similarly small mappings

Reply:
you mean something similar like this? https://github.com/AstroNvim/astrocommunity/blob/f2d802f9514e1b37f24289029451aa604075f931/lua/astrocommunity/motion/nvim-spider/init.lua#L1-L19

Reply:
More like(an example: 
```lua
{
      "AstroNvim/astrocore",
      opts = {
        mappings = {
          n = {
            ["<Leader>fe"] = { "<Cmd>Telescope file_browser<CR>", desc = "Open File browser" },
          },
          v = {
            ["<Leader>fe"] = { "<Cmd>Telescope file_browser<CR>", desc = "Open File browser" },
          },
        },
      },
},
```